constraint/pkg/regorewriter/regorewriter_test.go

```diff
@@ -495,3 +499,56 @@ func TestRegoRewriterErrorCases(t *testing.T) {
 		}
 	}
 }
+
+func TestRegoRewriterParseErrors(t *testing.T) {
+	testcases := []RegoRewriterTestcase{
+		{
+			name:      "fail to add base",
+			baseSrcs:  map[string]string{"invalid.rego": RegoSrc("foo", Body("something invalid"))},
+			wantError: true,
+		},
+		{
+			name:      "fail to add lib",
+			libSrcs:   map[string]string{"invalid.rego": RegoSrc("foo", Body("something invalid"))},
+			wantError: true,
+		},
+	}
+	for _, tc := range testcases {
+		t.Run(tc.name, tc.Run)
+	}
+}
+
+func TestRegoRewriterAddLibFromFs(t *testing.T) {
+	tests := []struct {
+		name    string
+		path    string
+		wantErr bool
```

Comment:
Create a base error for this failure mode with `errors.New()`. The function should wrap the base error with additional information, and then this test can test for the error with `errors.Is()`

This ensures we aren't just getting _some error_, but the particular error we're looking for.